packages/yaml/CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# @rollup/plugin-yaml Change Log
+
+## 2.0.0
+*2019-10-18*
+
+* Add `transform` option [#6](https://github.com/rollup/rollup/pull/6) (by @CharlesHolbrow)
```

Comment:
URLs in this file are wrong

Reply:
good catch. this is a holdover from the rollup-plugin-alias repo, so we'll fix this here but the original repo will still have the errors.